apps/full-site-editing/full-site-editing-plugin/editor-site-launch/src/launch-steps/privacy-step/index.tsx

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+import { Button } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import LaunchStep, { Props as LaunchStepProps } from '../../launch-step';
+import './styles.scss';
+
+const PrivacyStep: React.FunctionComponent< LaunchStepProps > = ( { onNextStep } ) => {
+	const handleContinue = () => {
+		onNextStep && onNextStep();
+	};
+
+	return (
+		<LaunchStep className="privacy-step">
+			<div className="launch-step__header">
+				<div className="launch-step__heading">
+					<h1 className="launch-step__title">{ __( 'Site privacy', 'full-site-editing' ) }</h1>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **23** times:
`translate( 'Site privacy' )` *ES Score: 11*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/plans-grid-fse-smart-component)